crates/hir-ty/src/chalk_db.rs

```diff
@@ -850,13 +848,6 @@ fn impl_def_datum(
     };
     let where_clauses = convert_where_clauses(db, impl_id.into(), &bound_vars);
     let negative = impl_data.is_negative;
-    debug!(
```

Comment:
This keeps popping up in logs when you set debug/trace for any specific module. I am not sure why (likely a bug in our logging infra somehow), but it is rather annoying